src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  margin-bottom: 24px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+
+  &__star {
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+    background: no-repeat url(images/star-active.svg);
+    background-position: center;
+
+    &--color_grey {
+      background: no-repeat url(images/star.svg);
+      margin-right: 0;
+      background-position: center;
+    }
+  }
```

Comment:
pay attention on [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) how to set styles for stars
![Screenshot from 2023-07-09 18-13-06](https://github.com/mate-academy/layout_catalog/assets/33029864/6dc52945-7816-400f-be63-b266c1dcd346)